__tests__/app.test.js

```diff
@@ -365,3 +365,40 @@ describe("Task 8 - DELETE /api/comments/:comment_id", () => {
     return request(app).delete("/api/comments/").expect(404);
   });
 });
+
+describe("Task 10 - GET /api/users", () => {
+  test("should return with an array", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((response) => {
+        const users = response.body;
+        expect(users).toBeInstanceOf(Array);
+      });
+  });
+  test("should return with an array of users which has a length greater than 0", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((response) => {
+        const users = response.body;
+        expect(users.length).not.toBe(0);
+      });
+  });
+  test("should return with an array of users who's objects match the required properties", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((response) => {
+        const users = response.body;
+        expect(users.length).not.toBe(0);
+        users.map((user) => {
+          expect(user).toMatchObject({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String),
+          });
+        });
+      });
```

Comment:
Very good. You can do all of these assertions with the one get request though. I.e. it doesnt need to be split between 3 tests 👍